grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -424,22 +424,22 @@ private void invokeOnCancel() {
         }
     }
 
-    void onError(Throwable t) {
+    public void onError(Throwable t) {
```

Comment:
```suggestion
    protected void onError(Throwable t) {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!